deployer/src/deployment/run.rs

```diff
@@ -549,6 +550,7 @@ async fn provision(
                 *bytes = serde_json::to_vec(&ShuttleResourceOutput {
                     output: new_secrets.clone(),
                     custom: shuttle_resource.custom,
+                    state: ResourceState::Ready
```

Comment:
Might not be worth changing this model on alpha. Consider using a `Beta` copy of it, but we also have time to re-think how we record resources.